app/Http/Controllers/DirectCostEstimateController.php

```
@@ -0,0 +1,36 @@
+        $data = ResourceItem::with('project', 'task')
+            ->when($task_id, fn ($query) => $query->filterByTaskId($task_id))
+            ->when($resource_type, fn ($query) => $query->filterByResourceType($resource_type))
+            ->get();
+        return DirectCostEstimateResource::collection($data)
+            ->additional([
+                'success' => true,
+                'message' => 'Direct Cost Estimates retrieved successfully',
+            ]);
+    }
+
+    public function show($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)